### ISSUE TITLE
Slight tweak to interface

### ISSUE THREAD

**Issue by midknight41** [midknight41/lab-suite#1]
The current interface is like this:
```js
import * as suite from "lab-suite";

export default suite.declare((lab, variables) => {

  // Do validation stuff here.

  const {
    SERVICE,
    SERVICE_CALL,
    SERVICE_INPUT,
    SERVICE_RESULT} = variables;

  lab.experiment(`The ${SERVICE_CALL} method`, () => {

    lab.test("succeed when get called with good parameters", done => {

      const service = new SERVICE();

      return service[SERVICE_CALL](...SERVICE_INPUT)
        .then(result => {
          expect(result).to.equal(SERVICE_RESULT);
        });
    });
  });
});
```
...but I think this is better. It makes the template variable definition a bit more explicit. Opinions?

```js
import labSuite from "lab-suite";

const suite = labSuite.create();

suite.expect("SERVICE").to.be.a.function();
suite.expect("SERVICE_CALL").to.be.a.string();
suite.expect("SERVICE_INPUT").to.be.an.array();
suite.expect("SERVICE_RESULT").to.be.a.string();

suite.declare((lab, variables) => {

  const {
    SERVICE,
    SERVICE_CALL,
    SERVICE_INPUT,
    SERVICE_RESULT
  } = variables;

  lab.experiment(`The ${SERVICE_CALL} method`, () => {

    lab.test("succeeds when get called with good parameters", done => {

      const service = new SERVICE();

      return service[SERVICE_CALL](...SERVICE_INPUT)
        .then(result => {
          expect(result).to.equal(SERVICE_RESULT);
        });
    });
  });
});

export default suite;
```
@b0mba @iancrowther @jmac105 @DamianStanger

**Comment by jmac105** [comment]
Agreed, I think it clearer to read initially. As long as you expect SERVICE_INPUT to be an array instead of a string :)

**Comment by midknight41** [comment]
Looks correct to me. ;-)

**Comment by midknight41** [comment]
shipped as v1.0.0
